lib/Gemfile

```diff
@@ -5,6 +5,7 @@ source 'https://rubygems.org'
 gem "bundler", "~>1.3"
 gem "rake",    "~>10.1"
 gem "iniparse"
+gem "winrm"
```

Comment:
This value is already in the vmdb/Gemfile, so will need to be removed from there if you are placing it here.  Also note that the vmdb/Gemfile value is locked to our fork of WinRM.